AnselemOdims/Book-A-Meal
# Ch tools setup 163971820

### What does this PR do?
Sets up the necessary tools and dependencies needed for the project

#### Description of Task to be completed?
- Install and setup Node/Express server-side framework 
- Install and setup ESLint for linting and Airbnb for the acceptable style guide
- Include a .eslintrc file
- Install and setup Babel to transpile code from ES6 or higher to ES5
- Include a .babelrc file
- Install the moment package for updating of time and date 
- Install the uuid package for creating unique IDs
- Install Nodemon for automatically restarting the server anytime a change is made
- Install and setup Mocha and Chai for unit testing of the program

#### How can it be manually tested
- Clone this repository
- cd into the directory
- Do git checkout ch-tools-setup-163971820
- Goto package.json to see all install packages

#### What are the relevant pivotal tracker stories?
[ Finishes #163971820](https://https://www.pivotaltracker.com/n/projects/2241503")
